app/views/layouts/cms.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en" class="govuk-template ">
   <head>
-    <title>Help for early years providers (editing content)</title>
+    <title>Editing Content | Help for early years providers | GOV.UK</title>
```

Comment:
https://www.gov.uk/service-manual/design/writing-for-user-interfaces#headings-and-title provides example, 'Where do you live? - register to vote - GOV.UK'.  Note: The use of hyphens and not pipes, to separate h1 from service name, and to separate service name from 'GOV.UK'